x/ethbridge/types/msgs.go

```diff
@@ -224,12 +224,12 @@ func ComputeProphecyID(cosmosSender string, sequence uint64, ethereumReceiver st
 	bytes, _ := arguments.Pack(
```

Comment:
I think this'll fail with "argument count mismatch", since argument at L197 wasnt updated

Reply:
Great catch Michael. I have updated that section accordingly as well as fixed the unit test for msgs.go